src/openklant/components/klantinteracties/api/schema.py

```diff
@@ -7,14 +8,71 @@
     KANAAL_PARTIJ,
 )
 
-# TODO: write a propper description
-description = """
-Description WIP.
+DESCRIPTION = _(
+    """
+**Warning: Difference between `PUT` and `PATCH`**
+
+Both `PUT` and `PATCH` methods can be used to update the fields in a resource,
+but there is a key difference in how they handle required fields:
+
+* The `PUT` method requires you to specify **all mandatory fields** when updating a resource.
+If any mandatory field is missing, the update will fail. Optional fields are left unchanged if they are not specified.
+
+* The `PATCH` method, on the other hand, allows you to update only the fields you specify.
+Some mandatory fields can be left out, and the resource will only be updated with the provided data,
+leaving other fields unchanged.
 """
+)
+PARTIJ_IDENTIFICATOR_DESCRIPTION_CREATE = _(
+    """
+**Warnings:**
+
+Handles `partijIdentificatoren` creation with atomicity guarantees.
+
+- If the `UUID` is provided in the `PartijIdentificator` object,
+the endpoint will treat it as an update operation for the existing `PartijIdentificator`,
+applying the provided data and linking the parent `Partij` to the new one created.
+- If the `UUID` is **not** specified, the system will create a new
+`PartijIdentificator` instance respecting all uniqueness constraints.
+    """
+)
+
+PARTIJ_IDENTIFICATOR_DESCRIPTION_UPDATE = _(
+    """
+**Warnings:**
+
+Handles `partijIdentificatoren` updates with atomicity guarantees.
+
+- If the `UUID` is provided in the `PartijIdentificator` object,
+the system will update the specified instance with the new data.
+- If the `UUID` is **not** specified, the system will `DELETE` all `PartijIdentificator`
+objects related to the parent and `CREATE` new ones with the new passed data.
```

Comment:
Might be good to explicitly mention the case of multiple PI where some have UUIDs and some don't here as well (like you explained here https://github.com/maykinmedia/open-klant/pull/356#discussion_r1986954344)